Generator/DoctrineFormGenerator.php

```diff
@@ -88,6 +89,7 @@ public function generate(BundleInterface $bundle, $entity, ClassMetadataInfo $me
             // Add 'setDefaultOptions' method with deprecated type hint, if the new 'configureOptions' isn't available.
             // Required as long as Symfony 2.6 is supported.
             'configure_options_available' => method_exists('Symfony\Component\Form\AbstractType', 'configureOptions'),
+            'get_name_required' => Kernel::MAJOR_VERSION == 2 && Kernel::MINOR_VERSION < 8,
```

Comment:
I would check for existence of the `getBlockPrefix()` method in the `AbstractType` class instead (version of the HttpKernel and Form component can differ).